client/src/pages/CreateAction.js

```diff
@@ -1,25 +1,154 @@
 import "./CreateAction.css";
 import Card from "../components/Card";
-import HeaderGoForward from "../components/Card_components/HeaderGoForward";
+import HeaderActionGoForward from "../components/Card_components/HeaderActionGoForward";
 import SpeechInput from "../components/SpeechInput";
-import { useParams } from "react-router";
+import { useParams, useHistory } from "react-router";
+import { useState, useEffect } from "react";
 import FooterSubmit from "../components/Card_components/FooterSubmit";
 
 export default function CreateCase() {
+  /** VARIABLE SET UP */
+  const reminderArray = JSON.parse(localStorage.getItem("newEntry"));
```

Comment:
You need to be careful here. What happens when there is no `newEntry` in the localStorage?
You could use the logical OR the check that.
```js
JSON.parse(localStorage.getItem("newEntry")) || [];
```
This reads like this: When the left side is not true, return the left side.

Reply:
The createCase will only be loaded once the createCase has run. CreateCase opens up the new Entry. That's why it shouldn't matter in my opinion

Reply:
The Create Action can only be accessed after a CreateCase that should logically prevent someone to create an action without a newEntry

Reply:
Going here: https://neue-fische-capstone-in-cased-git-feat-createentry-me-jterhorst.vercel.app/create/1 without having anything typed before breaks the app

![grafik](https://user-images.githubusercontent.com/54703643/130686527-51c71ad1-fe6c-4fbd-975e-c66cac72bd74.png)